docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -68,6 +68,23 @@ Set the systemd unit name prefix for pods. The default is *pod*.
 
 Set the systemd unit name separator between the name/id of a container/pod and the prefix. The default is *-*.
 
+#### **--wants**=*dependency_name*
+
+Set the systemd unit wants (`Wants=`), that this service is (weak) dependent on. This option may be specified more than once.
+This option does not influence the order in which services are started or stopped.
+
+The wants *network-online.target* is default and couldn't be removed or rewritten.
```

Comment:
> couldn't be removed or rewritten

As above. I don't understand what that means. Does it mean that the specified values are _appended_ but do not change the defaults?

Reply:
Thank you. I updated the doc trying to make it clear as possible